content/docs/dvclive/get-started.md

```diff
@@ -116,51 +78,98 @@ for epoch in range(NUM_EPOCHS):
     live.next_step()
 ```
 
-## Outputs
+</tab>
+</toggle>
+
+Check the [ML Frameworks](/doc/dvclive/api-reference/ml-frameworks) page for
+more details and other supported frameworks.
+
+If you prefer to use DVCLive directly, check the overview of the
+[Python API](/doc/dvclive/api-reference).
 
-After you run your training code, you should see the following content in the
-project:
+### Run the script
+
+Once you have added DVCLive to our python code, you can run the script as you
+would usually do:
 
 ```dvc
-$ tree
-├── dvclive
-│   ├── images
-│   │   └── img.png
-│   ├── params.yaml
-│   ├── plots
-│   │   └── roc.json
-│   └── scalars
-│       └── acc.tsv
-└── dvclive.json
+$ python train.py
+```
+
+DVCLive will generate a [report](/doc/dvclive/outputs#report) in
+`dvclive/report.html` containing all the logged data. It will be automatically
+updated during training on each `step` update:
+
+![HTML report](/img/dvclive-html.gif).
+
+You can find more details about the structure of files generated by DVCLive in
+the [Output folder structure](/doc/dvclive/outputs).
+
+## Set up DVC
+
+To build on the previous section, you can optionally wrap our code with DVC to
+run [DVC experiments](/doc/user-guide/experiment-management/) and enable new
+ways to compare and visualize our experiments.
+
+You can find below how an example `dvc.yaml` containing a single sage `train`,
+that runs the script from previous steps and tracks the
+[DVCLive outputs](/doc/dvclive/outputs) as `dvc metrics` and `dvc plots`:
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - train.py
+    metrics:
+      - dvclive.json:
+          cache: false
+    plots:
+      - dvclive/scalars:
+          cache: false
 ```
 
-The contents of the `dvclive` folder and `dvclive.json` will vary depending on
-the type of data you have logged and whether you have updated the step value or
-not.
+## Run experiments
 
-See `Live.log()`, `Live.log_param()`/`Live.log_params()`, `Live.log_image()` and
-`Live.log_plot()` for more details.
+You can now
+[run experiments](/doc/user-guide/experiment-management/running-experiments)
+with `dvc exp run`:
+
+```dvc
+$ dvc exp run
+Running stage 'train':
+> python train.py
+...
+```
 
-### Metrics report
+## Compare and visualize experiments
 
-If and when `step` is updated, DVCLive generates or updates a report in
-`dvclive/report.{format}` which will contain all the logged data.
+After following the above steps, you have enabled different ways of monitoring
+the training, comparing, and visualizing the experiments:
 
-![](/img/dvclive-html.gif)
+### DVC
 
-The `format` can be HTML) or Markdown depending on the value of the `report`
-argument passed to [`Live()`](/doc/dvclive/api-reference/live#parameters).
+You can use `dvc exp show` and `dvc plots` to compare and visualize metrics,
```

Comment:
Link to https://dvc.org/doc/user-guide/experiment-management/comparing-experiments and/or https://dvc.org/doc/user-guide/experiment-management/visualizing-plots?